### DISCUSSION TITLE
To screen or not to screen? Most people add a screen or not?

### DISCUSSION THREAD

**Discussion by DSew20** [outdoorbits/little-backup-box#302] [General]
Just curious - do most people add some form of screen / buttons or just run bare and use the web interface on their phone?

Just looking to rebuild my set-up and thinking maybe simplifying and using web only. The idea os simplicity sort of appeals and maybe more solid / stable / robust?

**Comment by outdoorbits** [comment]
I have no idea about this statistic. I think it's a question about how you use it.
I enjoy to see the progress on the display. And I use the IP info at the display.
Also I sometimes need the buttons, but only when traveling. After connecting to a new WiFi, sometimes it does not allow to communicate between WiFi clients, so I can't get access my lbb in this WiFi area. By display menu I can wipe the WiFi settings. (Having a second WiFi adapter keeps comitup hotspot open and solves the problem without buttons.)
